### DISCUSSION TITLE
-.

### DISCUSSION THREAD

**Discussion by ghost** [WeAreMahsaAmini/Core#256] [Ideas]

**Comment by SinaKarimi7** [comment]
I believe this will be very useful for most Persian Twitter users. I remember we had a service similar to this a few years ago. This tool is called "RostamBot" and @hameds created it. The whole story was told in [this thread](https://twitter.com/Hamed/status/1562783411177078784?s=20&t=0O5uaYg5v2WfSfiaj3XHBA).

"RostamBot" isn't available now but It would be great if we could continue/refactor that project and start this service again.

**Reply by SinaKarimi7** [reply]
@syavash What's your opinion?

**Comment by SinaKarimi7** [comment]
Additionally, we have an extension for Chrome called "[Bot Sight](https://chrome.google.com/webstore/detail/botsight/kahiiipphdmmkkamddaiiddmidmhbbil)" that shows Twitter account similarity percentages to bots.

Although it doesn't recognize cyber forces properly, it can inspire us to develop better user experiences.

**Comment by pooyakn** [comment]
It's a good idea. I'd be happy to work on it if nobody has started yet.
If so, I will write a POC by the end of this week to determine how accurate this idea is.
In time we can add more types of checks to it. After POC, we can define the first version requirements.

@Mojionix I'd appreciate it if you could share your database.

**Reply by pooyakn** [reply]
You can upload the DB on google drive or mega.co.nz and share it here or @pooyakn on telegram.
About code design, for sure. I read your first port, and as I understand, you say that having a few thousand verified Arzeshi accounts, we can check who follows these accounts and who these accounts follow, and based on that, estimate a risk factor.
What I was thinking about was creating a risk index based on several factors. One of them can be what you proposed. Other factors like Ghasem check and registration date can be helpful, too.

**Comment by ghost** [comment]


**Reply by ma21ar** [reply]
I'm also interested to work on this, I also have few ideas that might help. Let's create a discord room for this so we can collaborate more efficiently.

**Reply by pooyakn** [reply]
Hi.
Sorry for the delay.
Everyone is welcome to start working on this, and I think we can still improve the idea.
But I'm still working on it, and I'll push an MVP soon. I was busy with different things this week.
I'm in the middle of a transition in my work, and it was an abrupt decision, but no excuses. I'll get back to it ASAP.
I also found some people to help with other aspects, like if we want to develop an app for it.

**Comment by pachol1401** [comment]
Hello All,
I am so interesting in this idea and topic. So, I am searching for several days to find what we can do.
I found an App [ichbinhier_twittertools] (https://github.com/pkreissel/ichbinhier_twittertools) which is based on the Django Python Web-Framework. This app can log in to our twitter account and do the block function. But, it should be developed to be able to use it as a chrome extension and use our database to block the accounts in the database. @ma21ar @Mojionix @syavash 

However, I have also another suggestion until the day that we have own app.
There is a chrome extension which developed based on [ichbinhier_twittertools] and can block all accounts which liked on specific tweet. It called [Likers Blocker] (https://chrome.google.com/webstore/detail/likers-blocker/melnbpmfhaejmcpfflfjmchondkpmkcj?hl=en-GB).
I took some screen shoots to show how it works. Order from 1 to 9. I hope it would be helpful:

![1](https://user-images.githubusercontent.com/115351964/194766163-11db4cd0-e473-4081-b13d-a453b15402dd.jpg)
![2](https://user-images.githubusercontent.com/115351964/194766164-748c3a3a-95b9-48a1-8ac4-2752ad5db234.jpg)
![3](https://user-images.githubusercontent.com/115351964/194766165-dd63ba4b-ce55-429d-ba60-38b0d3dee58d.jpg)
![4](https://user-images.githubusercontent.com/115351964/194766166-a0b9c2a3-b4db-4ded-8247-14309952f4cc.jpg)
![5](https://user-images.githubusercontent.com/115351964/194766167-640080bb-c631-4b41-bb8f-aea38624ca4a.jpg)
![6](https://user-images.githubusercontent.com/115351964/194766168-7061e204-b981-4017-8513-8e112d24b296.jpg)
![7](https://user-images.githubusercontent.com/115351964/194766170-d75aaaf2-d3e1-4f54-a697-5aadf1862785.jpg)
![8](https://user-images.githubusercontent.com/115351964/194766172-5992f220-4ca0-4504-bea5-f72d4dac162d.jpg)
![9](https://user-images.githubusercontent.com/115351964/194766174-cd69ed40-26a0-4d08-8198-447c658f78a9.jpg)

**Comment by omegasoft7** [comment]
I had a similar idea, but i was thinking about having an app or s service that profiles can be reported and then accounts that are connected to the bot will block that profile if admin approves it. In this way you don't need to use Twitter web anymore and the bot will automatically and daily update your block list
